src/main/java/studentdirectory/enums/SortOrder.java

```diff
@@ -0,0 +1,13 @@
+package studentdirectory.enums;
+
+public enum SortOrder {
```

Comment:
enum naming?

Reply:
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums
https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN (this is for dot net)
In these websites it is discussed to not use enum suffix in naming

Reply:
add Type suffix